be/src/exec/pipeline/pipeline_driver_queue.h

```diff
@@ -123,11 +147,21 @@ class QuerySharedDriverQueueWithoutLock : public FactoryMethod<DriverQueue, Quer
 
 private:
     void _put_back(const DriverRawPtr driver);
+    // When the driver at the i-th level costs _level_time_slices[i],
+    // it will move to (i+1)-th level.
+    int _compute_driver_level(const DriverRawPtr driver) const;
 
+private:
     static constexpr size_t QUEUE_SIZE = 8;
     static constexpr double RATIO_OF_ADJACENT_QUEUE = 1.2;
+    // The time slice of the i-th level is (i+1)*LEVEL_TIME_SLICE_BASE ns,
+    // so when a driver's execution time exceeds 0.2s, 0.6s, 1.2s, 2s, 3s, 4.2s, 5.6s, and 7.2s,
+    // it will move to next level.
+    static constexpr int64_t LEVEL_TIME_SLICE_BASE_NS = 200'000'000L;
```

Comment:
why we choose 200ms?

Reply:
It's just an empirical value. The yield time of driver is 100ms, so maybe let a driver run 2 times on the highest priority queue.

Reply:
Will it overfit to this workload, and regressed in other workloads?

Reply:
@mofeiatwork  The yield time of driver is fixed 100ms for all the scenarios. Let me do most benchmark tests on SSB, TPC-DS, and TPC-H dataset.